blogposts/2021/release-3.28.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Sourcegraph 3.28 release"
+publishDate: 2021-05-20T10:00-07:00
+description: "Sourcegraph 3.28 release"
+tags: [blog, release]
+slug: "release/3.28"
+published: true
+changelogItems:
+  # Added
+  - description: "Added `select:commit.diff.added` and `select:commit.diff.removed` for `type:diff` search queries. These selectors return commit diffs only if a pattern matches in `added` (respectively, `removed`) lines."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20328
+    category: Search
+  - description: "Additional language autocompletions for the `lang:` filter in the search bar."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20535
+    category: Search
+  - description: "Steps in batch specs can now have an `if:` attribute to enable conditional execution of different steps."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20701
+    category: Batch Changes
+  - description: "Extensions can now log messages through `sourcegraph.app.log` to aid debugging user issues."
+    url: https://github.com/sourcegraph/sourcegraph/pull/20474
+    category:
+  # Changed
+  - description: "User and site credentials used in Batch Changes are now encrypted in the database if encryption is enabled with the `encryption.keys` config."
+    url: https://github.com/sourcegraph/sourcegraph/issues/19570
+    category: Batch Changes
+  - description: "All Sourcegraph images within [deploy-sourcegraph](https://github.com/sourcegraph/deploy-sourcegraph) now specify the registry. Thanks! @k24dizzle #2901."
+    url: "https://github.com/sourcegraph/deploy-sourcegraph/pull/2901"
+    category:
+  - description: Default reviewers are now added to Bitbucket Server PRs opened by Batch Changes.
+    url: https://github.com/sourcegraph/sourcegraph/pull/20551
+    category: Batch Changes
+  - description: Only site admins can now list users on an instance.
+    url: https://github.com/sourcegraph/sourcegraph/pull/20619
+    category: Admin
+  - description: "Repository permissions can now be enabled for site admins via the `authz.enforceForSiteAdmins` setting."
```

Comment:
@christinaforney - Can you provide context on all of the admin updates. What is the implication for users? Is it worth mentioning?